tests/script-test-consensus.ts

```diff
@@ -0,0 +1,60 @@
+import { HandlerConstructorConfig, RPCHandler } from "../dist";
+import { RequestPayload } from "../dist/types/rpc-service";
+
+/**
+ * A test script to ensure that the module can be imported and used correctly
+ * This script is not meant to be run in the test suite
```

Comment:
actually this is test and all your script-***.ts files. These are e2e tests. Move them to tests/e2e folder. At the end of each test, instead of console.log, use expect().